ingest.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "Starting ingest..."
+source ./venv/Scripts/activate
```

Comment:
```suggestion
source ./venv/bin/activate
```
Why `Scripts`? Same thing in the other places.

Reply:
Because I did this on a windows pc haha. Sorry, fixed it.

Reply:
This also needs to change in the other scripts too.

Reply:
I changed it